### ISSUE TITLE
.Net client throw ArgumentNullException from JToken.FromObject when Invoke hub method with paramater value null

### ISSUE THREAD

**Issue by Xiaohongt** [SignalR/SignalR#1655]
e.g. 
```csharp
//use the asp.net sample url
var hubConnection = new HubConnection("http://localhost:40476/");
var myHub = hubConnection.CreateHubProxy("shapeShare");
hubConnection.Start().Wait();
myHub.Invoke("changeUserName", "test", null).Wait();
```

Results:
Unhandled Exception: System.ArgumentNullException: Value cannot be null.
Parameter name: o
   at Newtonsoft.Json.Linq.JToken.FromObjectInternal(Object o, JsonSerializer jsonSerializer)
   at Newtonsoft.Json.Linq.JToken.FromObject(Object o, JsonSerializer jsonSerializer)
   at Microsoft.AspNet.SignalR.Client.Hubs.HubProxy.Invoke[T](String method, Object[] args) in e:\dd\SignalR_Dev_1.1_2\S
ignalR\src\Microsoft.AspNet.SignalR.Client\Hubs\HubProxy.cs:line 89
   at Microsoft.AspNet.SignalR.Client.Hubs.HubProxy.Invoke(String method, Object[] args) in e:\dd\SignalR_Dev_1.1_2\Sign
alR\src\Microsoft.AspNet.SignalR.Client\Hubs\HubProxy.cs:line 70

**Comment by Inverness** [comment]
Why does this error still exist? It's a simple fix by replacing JToken.FromObject(), which does a null check, with JValue.CreateNull() if the incoming argument is null.

**Comment by moozzyk** [comment]
Fixed in 389427e88eaff56bd9e40bcf7521af4d1c603f07
